articles/10-ways-to-contribute-to-geeksblabla.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "10 Ways to Contribute to GeeksBlabla and Make a Lasting Impact"
+tags: ["community", "GeeksBlabla", "contributions", "tech"]
+keywords: ["GeeksBlabla", "contribute", "Moroccan tech community"]
+pubDatetime: 2024-12-01
+description: "Explore how you can contribute to GeeksBlabla, from sharing feedback to joining the core team, and make a meaningful impact in the Moroccan tech community."
+---
+
+**"How can I contribute to the GeeksBlabla community?"** This is one of the most common questions we receive, and this blog post is here to answer it.
+
+Contributing to GeeksBlabla goes beyond being a core member—it’s about making an impact in ways that resonate with you, whether it’s by sharing feedback, suggesting topics, or being a guest on an episode. Some contributions, like giving a talk at BlablaConf, create a ripple effect that continues to inspire others long after.
+
+Whether you're a student exploring the tech world or a seasoned professional looking to give back, this guide will show you how you can make a difference while growing alongside the community. Let’s get started!
+
+## 1. Provide Feedback
+
+- **Impact:** ⭐⭐⭐
+- **Difficulty:** ⭐
+- **Time Investment:** ⭐
+
+Constructive feedback helps us improve. Whether it’s about our content, events, or initiatives, your input is invaluable in shaping the future of GeeksBlabla.
+
+Taking a few minutes to share your thoughts can lead to big improvements and better experiences for everyone involved.
```

Comment:
We can include the ability for people to suggest new concepts that we can work on.I know it's somewhat included here as part of the feedback, but I would love to make it explicit. The most useful aspect is that people are suggesting new concepts and projects for the community to work on.